src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -492,8 +492,31 @@ class HttpGoogleServicesDAO(
       list.getBillingAccounts.toSeq
     }) {
       case e: HttpResponseException if e.getStatusCode == StatusCodes.Forbidden.intValue =>
-        throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden, e.getContent))
+        val INSUFFICIENT_SCOPES = "Request had insufficient authentication scopes."
+        val googlyError = getGoogleErrorMessage(e)
+        if( googlyError == INSUFFICIENT_SCOPES ) {
+          throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden, s"""{"requiredScopes" : [%s]}""".format(billingScopes.head) ))
+        } else {
+          throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden, googlyError))
+        }
+
+    }
+  }
+
+  def getGoogleErrorMessage(exc: HttpResponseException): String = {
```

Comment:
> All Failures can be part of the default != INSUFFICIENT_SCOPES case above.

Not technically true because a different error message is not semantically equivalent to a parsing error, but I'll fiddle with it.